### PR TITLE
Align emoji reactions across multiple rows.

### PR DESCRIPTION
**Pull request by jackrzhang**
Addressing #2608.

Before:
image

Now:
image

### CODE REVIEW

File: src/webview/css/css.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ ul {
 }
 .reaction {
   color: rgba(127, 127, 127, 1);
-  padding: 4px;
+  padding: 4px 5.5px;
```
Comment (borisyankov) [inline src/webview/css/css.js]: Hmm, 5.5px is even more magical than a magic number ;)

File: src/webview/html/messageReactionAsHtml.js [hunk 2]
```diff
@@ -19,5 +19,5 @@ export default (
     realmEmoji[reaction.name]
       ? getRealmEmojiHtml(realmEmoji[reaction.name])
       : emojiMap[reaction.name]
-  } ${reaction.count}
+  }&#160;${reaction.count}
```
Comment (borisyankov) [inline src/webview/html/messageReactionAsHtml.js]: Pretty good fix!

Comment (jackrzhang) [reply src/webview/html/messageReactionAsHtml.js]: No particular reason - agreed that &nbsp would be more clear. I'll change that :)

File: src/webview/css/css.js [hunk 3]
```diff
@@ -312,13 +312,11 @@ ul {
 }
 .reaction {
   color: rgba(127, 127, 127, 1);
-  padding: 4px;
```
[the remark this answers is not in the document]
Comment (jackrzhang) [reply src/webview/css/css.js]: > Visually, there seems to be more padding on the right than on the left. This is present already in your "before" screenshot, but I think it becomes more conspicuous after.

Will fix that :)

> The last reaction in the list is an exception to this; and in your "after" screenshot, also the last in the first row. The last in each row in your "after" look pretty great, I think, but the contrast with the others is jarring (particularly where that final octopus is lined up with the "up and to the right" above it, except the right border which doesn't line up.)

I think that's some peculiarity due to the fact that spans have an `inline` display value - it seems like the last character of a line has a little less automated padding after it than characters within a line, which is why the final octopus isn't lining up. If you look closely, you'll see that the tomato emoji reaction is also shorter in width than the others. Setting the display value to `inline-block` (and readjusting some layout values) should fix that.

**Comment by gnprice** [comment]
Merged -- thanks again!

I made a few tweaks to the commit messages; take a look.

**Comment by jackrzhang** [comment]
👍

**Comment by gnprice** [comment]
Err, *now* actually merged.

**Pull request closed without merging**